packages/microcosm-www-next/src/components/description.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+export default class Description extends React.Component {
+  componentDidMount() {
+    this.el = document.getElementById('fade-in-text')
+    this.listenForAnimationEnd(this.el)
+  }
+
+  listenForAnimationEnd(el) {
+    el.addEventListener('animationend', () => {
+      el.classList.remove('fade-in')
+    })
+  }
+
+  componentWillUpdate() {
+    this.el.classList.add('fade-in')
```

Comment:
when the description updates with new data, it will add .fade-in which triggers an animation. when that animation ends, it will trigger the event listener on line 10 to remove the class